dashboard/src/reducers/cluster.ts

```diff
@@ -123,6 +125,22 @@ const clusterReducer = (
           },
         };
       }
+    case getType(actions.config.receiveConfig):
+      // Initialize the additional clusters when receiving the config.
+      const clusters = {
+        default: state.clusters.default,
+      };
+      const config = action.payload as IConfig;
+      config.featureFlags.additionalClusters.forEach(cluster => {
```

Comment:
small fail-safe in case additionalClusters is not defined (like in a local config file): `config.featureFlags.additionalClusters &&  config.featureFlags.additionalClusters.forEach...`

Reply:
We can use `config.featureFlags.additionalClusters?.forEach` . Added a failing test and fixed with that (had to delete the property from a valid `IConfig` to stop ts from complaining :)